pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -184,6 +184,17 @@ type ShootStatus struct {
 	// managed to change the hibernation settings of the cluster
 	// +optional
 	LastHibernationTriggerTime *metav1.Time `json:"lastHibernationTriggerTime,omitempty" protobuf:"bytes,17,opt,name=lastHibernationTriggerTime"`
+	// LastMaintenance holds information about the last maintenance operation on the Shoot.
+	// +optional
+	LastMaintenance []LastMaintenance `json:"lastMaintenance,omitempty" protobuf:"bytes,18,opt,name=lastMaintenance"`
```

Comment:
```suggestion
	LastMaintenance []LastMaintenance `json:"lastMaintenances,omitempty" protobuf:"bytes,18,opt,name=lastMaintenances"`
```

(as well as all other suggestions from `core/types_shoot.go`; same for `v1beta1` package)